### ISSUE TITLE
Improve as_standard_surface ergonomy

### ISSUE THREAD

**Issue by dictoon** [appleseedhq/appleseed#1900]
Some suggestions on Slack:

1. @frang: Remove repetitions in input names (left: current; right: suggested):

![as_standard_surface](https://user-images.githubusercontent.com/321290/37246940-1d1120e0-24b3-11e8-9886-30e176137af9.png)

2. @juancarlosgzrz: Reorder sections:

Proposed order: Diffuse, Specular, Specular Anisotropy, Refraction, Coating, Subsurface, Translucency, Transparency, Incandescence.... move bump to the lowest section.

**Comment by juancarlosgzrz** [comment]
Also, choose between 'weight' and 'amount'. 

Replace points by spaces in name sections. i.e from Specular.Anisotropy to Speculat Anisotropy.

**Comment by dictoon** [comment]
> Also, choose between 'weight' and 'amount'.

Covered by https://github.com/appleseedhq/appleseed/issues/1901.

> Replace points by spaces in name sections. i.e from Specular.Anisotropy to Specular Anisotropy.

Good point.

**Comment by luisbarrancos** [comment]
We cannot replace points by spaces. The points denote the submenu separators are OSL metadata token. I.e, when building a UI, Specular.Anisotropy will generate Specular/Anisotropy groups/Tabs.
At least on Maya and Gaffer.

**Comment by luisbarrancos** [comment]
About the naming consistency, i agree with #1901, there is room for improvement here. Let's see first what we can do about standard surface since this was the 1st prototype and the system to layer closures is not yet in place. This will imply a rewrite of this shader, or at least some work which will invalidate the present UI imho.

**Comment by luisbarrancos** [comment]
@usakhelo the plugin can build nested groups/tabs? The dots are used for this, menu.submenu.

**Comment by dictoon** [comment]
> We cannot replace points by spaces.

I'm just talking about what we display in the user interface. We should be able to use spaces instead of points at display time (simply replacing `.` by ` ` when setting the UI widget's label).

**Comment by luisbarrancos** [comment]
@dictoon i mean there should be neither points nor spaces there. The plugin is not reading the page metadata token in order to build the menus and submenus dynamically. The points denote a menu.submenu in the string page metadata token. We have these in Maya and Gaffer, and in Max in terms of UI you also have this in the form of a roll-down main menu, and internal sub-sections, or should have.

**Comment by dictoon** [comment]
> The plugin is not reading the page metadata token in order to build the menus and submenus dynamically

Yes that's precisely what it's doing.

> in Max in terms of UI you also have this in the form of a roll-down main menu, and internal sub-sections, or should have

There is, and we do, but rollups aren't nested. Maybe they _could_ be nested from a technical standpoint, but I'm not sure users would expect this. I'm not away of any material with nested rollups.

**Comment by luisbarrancos** [comment]
@dictoon ok, but in that case i'm not sure what course of action is possible here from my part, since this formalism is required for nested menus in maya and gaffer.

**Comment by dictoon** [comment]
You're completely right, that part should have been a separate ticket in appleseed-max.

**Comment by luisbarrancos** [comment]
Closing this since it's related ticket https://github.com/appleseedhq/appleseed-max/issues/137 already solved this.

**Comment by dictoon** [comment]
@luisbarrancos Are you sure https://github.com/appleseedhq/appleseed-max/issues/137 solves this at all? This ticket is about redundant prefixes in parameter names and the order of parameter groups. Reopening until confirmed fixed.
